libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.ts

```diff
@@ -241,8 +240,6 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
       }
 
       this.#populateInputBoxHelpText();
-
-      this.selectedCountryChange.emit(this.#_selectedCountry);
```

Comment:
This is the crux of the change. The docs for the `selectedCountryChange` event state: "Emits a `SkyPhoneFieldCountry` object when the selected country in the country search input changes." However, this event was emitting when the `selectedCountry` input was set as well. Based on the documentation, this event should only fire when the user interacts with the country select, and NOT emit when the selected country is changed internally.